MeDrop/MeDrop/Sources/Creation/View/CreateInfoView.swift

```diff
@@ -9,15 +9,15 @@ import SwiftUI
 
 struct CreateInfoView: View { 
     @EnvironmentObject var myCards: EnvironmentData
-    @Environment(\.dismiss) private var dismiss
+   
+    @Binding var showingCreation: Bool
```

Comment:
```suggestion
    @Binding var isShowingCreation: Bool
```

변수/정수명을 보았을 때 타입을 바로 예상할 수 있도록 Bool 값은 의문문으로 하는 것을 추천드립니당 : )

Reply:
@unnnyong  감사합니다. 바로 반영하겠습니다.

Reply:
감사합니다!